src/main/java/edu/harvard/iq/dataverse/GlobalId.java

```diff
@@ -25,6 +27,7 @@
     public static final String HDL_PROTOCOL = "hdl";
     public static final String HDL_RESOLVER_URL = "https://hdl.handle.net/";
     public static final String DOI_RESOLVER_URL = "https://doi.org/";
+    public static final String ALLOWED_CHARACTERS = "^[A-Za-z0-9._/:\\-]*$";
```

Comment:
Looking at this more, I think this should actually be moved to our bundle.properties file. It'll help with later maintenance to have this config-y attribute stored there. Sorry I didn't catch this the first time through!